packages/cli-types/src/index.ts

```diff
@@ -120,6 +122,100 @@ export type ProjectConfig = {
   [key: string]: any;
 };
 
+export type NotFound = 'Not Found';
+type AvailableInformation = {
+  version: string;
+  path: string;
+};
+
+type Information = AvailableInformation | NotFound;
+
+export type EnvironmentInfo = {
+  System: {
+    OS: string;
+    CPU: string;
+    Memory: string;
+    Shell: AvailableInformation;
+  };
+  Binaries: {
+    Node: AvailableInformation;
+    Yarn: AvailableInformation;
+    npm: AvailableInformation;
+    Watchman: AvailableInformation;
+  };
+  SDKs: {
```

Comment:
is the idea that this is what will get asked of envinfo? if so, please make sure that info continues to report the windows properties we care about: Windows SDKs, VS installs, dev mode reg keys.